public/app/features/dashboard/api/v2.ts

```diff
@@ -31,9 +32,17 @@ export class K8sDashboardV2APIStub implements DashboardAPI<DashboardWithAccessIn
   async getDashboardDTO(uid: string, params?: UrlQueryMap) {
     const dashboard = await this.client.subresource<DashboardWithAccessInfo<DashboardV2Spec>>(uid, 'dto');
 
-    // For dev purposes only, the conversion should and will happen in the API. This is just to stub v2 api responses.
-    const result = ResponseTransformers.ensureV2Response(dashboard);
+    let result: DashboardWithAccessInfo<DashboardV2Spec> | DashboardDTO | undefined;
 
+    // TODO: For dev purposes only, the conversion should and will happen in the API. This is just to stub v2 api responses.
+    const backendAPINotReady = true; // this should be removed once the backend API is ready
+    if (backendAPINotReady) {
+      result = ResponseTransformers.ensureV2Response(dashboard);
+    } else {
+      result = dashboard;
+    }
```

Comment:
don't understand this, can't this be handled by ensureV2Response? This API should always return v2, unless it's configured to transform. This if statement is rettu much a comment :)

Reply:
good catch 👍🏾